samples/core/GetStarted/AspNetCore/EFGetStarted.AspNetCore.NewDb.Sqlite/Startup.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.HttpsPolicy;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+#region AddedUsings
```

Comment:
My preference is to add these as comments in the code and not break up the instructions. I think that appeals to better programmers.  Your approach might be better a few beginners.

Reply:
I decided to keep the `using` code block, as I think it's more convenient when following the tutorial to get the `using` statements with one click of the copy button on a code block, compared to scraping them out of code comments.  But the "requires using" code comments are helpful for people looking at the code who aren't following the tutorial, so I added those following the pattern I see in some of your other articles.